### ISSUE TITLE
"Inherited from" shows the name of the property, not the type

### ISSUE THREAD

**Issue by upta** [sebastian-lenz/typedoc#66]
For example, in this case the property comes from ng.IScope in the angular definitelyTyped declarations

```
{
	"id": 149,
	"name": "$id",
	"kind": 1024,
	"kindString": "Property",
	"flags": {
		"isExported": true
	},
	"type": {
		"type": "instrinct",
		"name": "number"
	},
	"inheritedFrom": {
		"type": "reference",
		"name": "$id"
	}
}
```
![capture](https://cloud.githubusercontent.com/assets/3310732/6233811/7f22614e-b68a-11e4-93fb-6c43056eb85a.PNG)

**Comment by sebastian-lenz** [comment]
I guess you didn't include the documentation of declaration files so the template falls back to print the name of the inherited symbol. I've added the name of the parent symbol so the output is more meaningful.

I'm more concerned to see the full path of <code>angular.d.ts</code> in your screenshot as this is an old and recurring bug. Do the source files of your project span across your whole hard drive?

**Comment by upta** [comment]
Yeah, I'm excluding the externals simply to cut down on the volume of stuff it has to process (at least while I'm actively developing my doc solution ;)

The full path bit could be because of how I have my project structure.  I have a "site" project that contains the typedoc pieces (using a grunt task) and a "core" project that's the actual code I'm documenting, which live in a folder structure like this:

```
root
-- docs
  -- site
-- modules
  -- core
```

And then in my "site" project I have a grunt task for running TypeDoc that looks like this:
```
typedoc:
{
    build:
    {
        options:
        {
            out: "json/typedoc",
            json: "json/docs.json",
            target: "ES5",
            includeDeclarations: "",
            mode: "file",
            externalPattern: "**/scripts/typings/**",
            excludeExternals: ""
        },
        src: [
            "../../modules/core/src/**/*.ts",
            "../../modules/core/scripts/**/*.ts"
        ]
    }
}
```

So I could see how you could easily end up with full-path info, but you know far better than I :)

I honestly didn't even notice the path until you mentioned it because I'm not actually using it in my own docs ;)

Thanks man!

**Comment by sebastian-lenz** [comment]
Version 0.3.1 is now on npm, is the issue resolved for you?

**Comment by sebastian-lenz** [comment]
I think this issue should be resolved with v0.3.1. Please message me if I'm wrong.
